.github/workflows/detect_version_bump.yml

```diff
@@ -21,10 +21,16 @@ jobs:
         run: |
           NEW_VERSION=$(echo ${{ github.head_ref }} | cut -d _ -f4 )
           echo "::set-output name=version::$NEW_VERSION"
+      - name: Is prerelease?
+        id: is-prerelease
+        run: |
+          IS_PRERELEASE=$([[ "${{ steps.get-new-version.outputs.version }}" == *-* ]] && echo true || echo false)
+          echo "::set-output name=result::$IS_PRERELEASE"
       - name: Create Release
         uses: actions/create-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
         with:
           tag_name: v${{ steps.get-new-version.outputs.version }}
           release_name: v${{ steps.get-new-version.outputs.version }} Release
+          prerelease: ${{ steps.is-prerelease.outputs.result }}
```

Comment:
I intend this to reference the `result` defined on [this line](https://github.com/wayfair-incubator/columbo/pull/119/files#diff-8853891674655968cc7a6b2e0b2fd0a53b96c4af0a7808283178d2f56533560cR28).